docs/_data/sidebars/mydoc_sidebar.yml

```diff
@@ -2,8 +2,8 @@
 
 entries:
 - title: sidebar
-  product: Latest
-  version: Version
+  product: Latest Version
```

Comment:
@vinothchandar : I need to make this change in this way to have it rendered properly : "Latest Version 0.5.0-incubating"